catalyst_voices/license_checker.yaml

```diff
@@ -0,0 +1,35 @@
+# cspell: words rxdart
+
+permittedLicenses:
+- MIT
+- Apache-2.0
+- Unicode-DFS-2016
+- BSD-3-Clause
+- BSD-2-Clause
+- BlueOak-1.0.0
+- Apache-2.0 WITH LLVM-exception
+- CC0-1.0
+
+packageLicenseOverride:
+  fuchsia_remote_debug_protocol: BSD-3-Clause
+  flutter_driver: BSD-3-Clause
+  flutter_localizations: BSD-3-Clause
+  flutter_test: BSD-3-Clause
+  flutter_web_plugins: BSD-3-Clause
+  integration_test: BSD-3-Clause
+  rxdart: Apache-2.0
+  vector_math: Apache-2.0
+  sky_engine: Apache-2.0
+  flutter_gen: MIT
+
+packageSourceOverride:
+  fuchsia_remote_debug_protocol: https://github.com/flutter/flutter/blob/3.19.5/LICENSE
```

Comment:
Pls sync it with latest flutter version, should be 3.22.1, check other places too